docs/evaluation/how_to_guides/prebuilt_evaluators.mdx

```diff
@@ -0,0 +1,195 @@
+import {
+  CodeTabs,
+  python,
+  typescript,
+} from "@site/src/components/InstructionsWithCode";
+
+# How to use prebuilt evaluators
+
+:::tip
+For the older version of this page, see [here](./use_langchain_off_the_shelf_evaluators_old).
```

Comment:
I'd remove this entirely. Anecdotally, havent heard that people use these. 

We should have one first class way of providing pre-built evaluators and I think referencing the old page here is confusing the story.

Reply:
Cool sounds good